playbulb-candle/index.html

```diff
@@ -14,7 +14,19 @@
   <body id="state">
     <div id="step1">
       <div id="title">PLAYBULB CANDLE DEMO</div>
-      <button id="connect" class="mdl-button mdl-js-button mdl-button--raised mdl-js-ripple-effect mdl-button--accent">CONNECT</button>
+      <button id="connect" class="mdl-button mdl-js-button
+    mdl-button--raised mdl-js-ripple-effect mdl-button--accent"
+      style="display:none">CONNECT</button>
+
+      <div id="no-bluetooth" class="no-bluetooth-card-square mdl-card
+                                    mdl-shadow--2dp" style="display:none">
+        <div class="mdl-card__title mdl-card--expand">
+          <h2 class="mdl-card__title-text">No Web Bluetooth</h2>
+        </div>
+        <div class="mdl-card__supporting-text">
+          Please enable the Web Bluetooth API at chrome://flags/#enable-web-bluetooth
```

Comment:
This demo should work for non-Chrome browsers too as they come up. So we should only mention chrome flags if we know we're on Chrome. We should point to the implementation status - as the platform and OS permissions matter too. https://github.com/WebBluetoothCG/web-bluetooth/blob/gh-pages/implementation-status.md And that page includes flags information. Perhaps generic text such as:

Web Bluetooth isn't currently enabled. Please check that your browser and device support it and have any necessary development flags set.